CMakeLists.txt

```diff
@@ -67,7 +67,8 @@ set (CMAKE_CONFIGURATION_TYPES "Debug;Release;Profile;Ship" CACHE STRING "" FORC
 if (NOT CMAKE_BUILD_TYPE)
     convert_list_to_string (config_types_string ${CMAKE_CONFIGURATION_TYPES})
```

Comment:
We can probably get rid of this line since `config_types_string` doesn't appear to be used anywhere.